ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -20,8 +20,10 @@
 tier2 = pytest.mark.tier2(value=2)
 tier3 = pytest.mark.tier3(value=3)
 tier4 = pytest.mark.tier4(value=4)
+tier4a = pytest.mark.tier4a
```

Comment:
You should keep the tier4 marker as they are still tier4 tests.

So please use:

```python
tier4a = compose(tier4, pytest.mark.tier4a)
```
Then you can still execute all tier4 tests with just `-m tier4` or  just new set of tier4 tests with `-m tier4a`
Or we can have some extra marker for this purpose as:
```python
first_cut = pytest.mark.first_cut
second_cut  = pytest.mark.second_cut
third_cut = pytest.mark.third_cut  # (maybe later)
```

Then it will be general for any other tiers and we can execute just with: `-m tier4 and first_cut` which execute first cut of tier4 tests only and we still have ability to trigger whole tier4 and as I said it's more general and can be used with other tier if needed.

Keep in mind that based on used markers it will be also reported to polarion with different test_run_id.

Reply:
@petr-balogh thanks for this explanation, will do that